src/v2/view-builder/views/ov/ChallengeOktaVerifyTotpView.js

```diff
@@ -16,29 +16,27 @@ const Body = BaseForm.extend(Object.assign(
       return loc('mfa.challenge.verify', 'login');
     },
 
-    isErrorCalloutCustomized(convertedErrors) {
-      const errorSummaryKeys = convertedErrors?.responseJSON?.errorSummaryKeys;
+    showCustomFormErrorCallout(error) {
+      const errorSummaryKeys = error?.responseJSON?.errorSummaryKeys;
+      let options;
       if (errorSummaryKeys && errorSummaryKeys.includes(OV_UV_ENABLE_BIOMETRIC_SERVER_KEY)) {
-        return true;
+        options = {
+          type: 'error',
+          className: 'okta-verify-uv-callout-content',
+          title: loc('oie.authenticator.app.method.push.verify.enable.biometrics.title', 'login'),
+          subtitle: loc('oie.authenticator.app.method.push.verify.enable.biometrics.description', 'login'),
+          bullets: [
+            loc('oie.authenticator.app.method.push.verify.enable.biometrics.point1', 'login'),
+            loc('oie.authenticator.app.method.push.verify.enable.biometrics.point2', 'login'),
+            loc('oie.authenticator.app.method.push.verify.enable.biometrics.point3', 'login')
+          ],
+        };
+        BaseForm.prototype.showMessages.call(this, createCallout(options));
```

Comment:
Does this mean that whoever overrides `showCustomFormErrorCallout` has to explicitly call `BaseForm.prototype.showMessages`?

Reply:
Fixed it..